misc/generate_accounts.sh

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bash
```

Comment:
A better script is needed to stream accounts from postgres. Rotem gave me a python script that exports accounts directly from algod, so I exported them to a file. My current tests are using a file to feed the validator tool.